/**:
  ros__parameters:
    clip_ball_xy: True
    filter_ball_xy: True

    xy_clip_limits: [-2.000, -1.000, 2.000, 1.711]
    xy_filter_limits: [-2.500, -1.500, 2.500, 2.211]